package.json

```diff
@@ -174,7 +179,7 @@
     }
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile && npm run tslint",
```

Comment:
Why are you removing the tslint?

Reply:
Oh my bad, that was intended to be temporary during development, fixed.